newdle/client/src/components/Footer.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {Trans} from '@lingui/macro';
+import {Grid} from 'semantic-ui-react';
+import client from '../client';
+import LanguageSelector from './common/LanguageSelector';
+import styles from './Footer.module.scss';
+
+export default function Footer() {
+  const [configuredLinks, loading] = client.useBackend(() => client.getFooterLinks(), []);
+
+  let links;
+  if (loading || configuredLinks === null) {
+    links = {};
+  } else {
+    links = configuredLinks;
+  }
+
+  return (
+    <footer className={styles.footer}>
+      <Grid columns={3}>
+        <Grid.Row verticalAlign="middle">
+          <Grid.Column computer={4} tablet={4} only="computer tablet">
+            <ProjectInfo />
+          </Grid.Column>
+          <Grid.Column mobile={16} tablet={8} computer={8}>
+            <ul className={styles.links}>
+              {Object.entries(links).map(([key, value]) => (
```

Comment:
By using a dict for the link we don't keep any specific order. Maybe better to use a list with `[title, link]` pairs?